src/exporter/ddprof_exporter.cc

```diff
@@ -140,49 +143,66 @@ DDRes ddprof_exporter_init(const ExporterInput *exporter_input,
   return ddres_init();
 }
 
-static void fill_tags(const UserTags *user_tags, const DDProfExporter *exporter,
-                      std::vector<ddprof_ffi_Tag> &tags_exporter) {
-  tags_exporter.push_back(ddprof_ffi_Tag{
-      .name = char_star_to_CharSlice("language"),
-      .value = string_view_to_CharSlice(exporter->_input.language)});
-
-  if (exporter->_input.environment) {
-    tags_exporter.push_back(ddprof_ffi_Tag{
-        .name = char_star_to_CharSlice("env"),
-        .value = char_star_to_CharSlice(exporter->_input.environment)});
+static DDRes add_single_tag_c(ddprof_ffi_Vec_tag &tags_exporter,
```

Comment:
Is this function really necessary since `string_view` has an implicit constructor from`const char*` ?

Reply:
Fixed